src/test/java/com/neo4j/docker/coredb/TestMounting.java

```diff
@@ -345,4 +348,23 @@ void shouldReownSubfilesToNeo4j() throws Exception {
             // if debug.log doesn't get re-owned, neo4j will not start and this test will fail here
         }
     }
+
+    @Test
+	void shouldLoadPluginsFromMountedFolder() throws Exception {
```

Comment:
The test name does not describe what's happening in the test.
It's testing that the bundled apoc plugin can be moved into the `/plugins` folder when something is mounted to `/plugins`. 

It's more of a test of the plugins downloading/moving infrastructure than anything to do with mounting. 

I think it should be in `TestPluginInstallation.java` instead.